extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -1192,32 +1212,28 @@ angular.module('oppia').component('svgFilenameEditor', {
         // This if condition is required to ensure that the size change is
```

Comment:
Please explain the branching in `onFillChange` for bezier draw mode and the else case in a comment.

Reply:
Done.